modules/@apostrophecms/modal/ui/apos/components/AposDocsManagerToolbar.vue

```diff
@@ -2,26 +2,26 @@
   <AposModalToolbar class-name="apos-manager-toolbar">
     <template #leftControls>
       <AposButton
+        v-if="!options.hideSelectAll"
         label="Select" :icon-only="true"
         :icon="checkboxIcon" type="outline"
         @click="$emit('select-click')"
       />
-      <AposTagApply :tags="applyTags" :apply-to="[]" />
-      <!-- TODO: trash component needs to be worked out with confirm, maybe separate into its own component -->
-      <AposButton
+      <!-- TODO: Return this delete button when batch updates are added -->
+      <!-- <AposButton
```

Comment:
This seemed easier than removing then later digging into history or figuring out how to add them back.